<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>xToolsResponserToolUiEditor</class>
 <widget class="QWidget" name="xToolsResponserToolUiEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>647</width>
    <height>474</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="2">
    <widget class="QPushButton" name="pushButtonOk">
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="pushButtonCancel">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Parameters</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="6" column="4">
       <widget class="xToolsAffixesComboBox" name="comboBoxResSuffix"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Description</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="checkBoxEnable">
        <property name="text">
         <string>Enable</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="label_15">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Start index</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QLabel" name="label_8">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Escape</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QCheckBox" name="checkBoxRefCrcEnable">
          <property name="text">
           <string>Append CRC data</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBoxRefCrcBigEndian">
          <property name="text">
           <string>Big endian byte order</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="10" column="3">
       <widget class="QLabel" name="label_18">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>End index</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QLabel" name="label_6">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Fromat</string>
        </property>
       </widget>
      </item>
      <item row="12" column="0" colspan="2">
       <widget class="QLabel" name="label_19">
        <property name="text">
         <string>Reference data</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="xToolsTextFormatComboBox" name="comboBoxRefFormat"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_5">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Format</string>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QLabel" name="label_16">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>End index</string>
        </property>
       </widget>
      </item>
      <item row="13" column="0" colspan="2">
       <widget class="QLineEdit" name="lineEditRefData"/>
      </item>
      <item row="8" column="4">
       <widget class="xToolsCrcAlgorithmComboBox" name="comboBoxResAlgorithm"/>
      </item>
      <item row="5" column="4">
       <widget class="xToolsAffixesComboBox" name="comboBoxResPrefix"/>
      </item>
      <item row="13" column="3" colspan="2">
       <widget class="QLineEdit" name="lineEditResData"/>
      </item>
      <item row="1" column="4">
       <widget class="xToolsResponseOptionComboBox" name="comboBoxOption"/>
      </item>
      <item row="6" column="3">
       <widget class="QLabel" name="label_12">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Suffix</string>
        </property>
       </widget>
      </item>
      <item row="8" column="3">
       <widget class="QLabel" name="label_14">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Algorithm</string>
        </property>
       </widget>
      </item>
      <item row="3" column="4">
       <widget class="xToolsTextFormatComboBox" name="comboBoxResFormat"/>
      </item>
      <item row="10" column="4">
       <widget class="QSpinBox" name="spinBoxResEnd">
        <property name="maximum">
         <number>1024</number>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="xToolsAffixesComboBox" name="comboBoxRefSuffix"/>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_13">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Algorithm</string>
        </property>
       </widget>
      </item>
      <item row="9" column="4">
       <widget class="QSpinBox" name="spinBoxResStart">
        <property name="maximum">
         <number>1024</number>
        </property>
       </widget>
      </item>
      <item row="5" column="3">
       <widget class="QLabel" name="label_11">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Prefix</string>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QSpinBox" name="spinBoxRefStart">
        <property name="maximum">
         <number>1024</number>
        </property>
       </widget>
      </item>
      <item row="2" column="3" colspan="2">
       <widget class="QLabel" name="label_4">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Response</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="xToolsAffixesComboBox" name="comboBoxRefPrefix"/>
      </item>
      <item row="8" column="1">
       <widget class="xToolsCrcAlgorithmComboBox" name="comboBoxRefAlgorithm"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_7">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Escape</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QLabel" name="label_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Reference</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_10">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Suffix</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="label_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Option</string>
        </property>
       </widget>
      </item>
      <item row="4" column="4">
       <widget class="xToolsEscapeCharacterComboBox" name="comboBoxResEscape"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_9">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Prefix</string>
        </property>
       </widget>
      </item>
      <item row="12" column="3" colspan="2">
       <widget class="QLabel" name="label_20">
        <property name="text">
         <string>Response data</string>
        </property>
       </widget>
      </item>
      <item row="10" column="1">
       <widget class="QSpinBox" name="spinBoxRefEnd">
        <property name="maximum">
         <number>1024</number>
        </property>
       </widget>
      </item>
      <item row="9" column="3">
       <widget class="QLabel" name="label_17">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Start index</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2" rowspan="12">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="xToolsTextFormatComboBox" name="comboBoxRefEscape"/>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEditDescription">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="7" column="3" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QCheckBox" name="checkBoxResCrcEnable">
          <property name="text">
           <string>Append CRC data</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBoxResCrcBigEndian">
          <property name="text">
           <string>Big endian byte order</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="11" column="3">
       <widget class="QLabel" name="label_21">
        <property name="text">
         <string>Delay(ms)</string>
        </property>
       </widget>
      </item>
      <item row="11" column="4">
       <widget class="QSpinBox" name="spinBoxResDelay">
        <property name="maximum">
         <number>10000</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>xToolsEscapeCharacterComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">xToolsEscapeCharacterComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>xToolsTextFormatComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">xToolsTextFormatComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>xToolsAffixesComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">xToolsAffixesComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>xToolsCrcAlgorithmComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">xToolsCrcAlgorithmComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>xToolsResponseOptionComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">xToolsResponseOptionComboBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
